pkg/azclient/policy/retry.go

```diff
@@ -35,7 +35,6 @@ import (
 func GetRetriableStatusCode() []int {
 	return []int{
 		http.StatusRequestTimeout,      // 408
-		http.StatusTooManyRequests,     // 429
```

Comment:
Do we still retry when hitting 429?

Reply:
no. the client will return error and let caller decide when to retry. throttle policy will read retry after header and reject any request if the client is throttled.